src/component/tooltip/tooltipMarkup.ts

```diff
@@ -32,20 +32,52 @@ import { getRandomIdBase } from '../../util/number';
 import Model from '../../model/Model';
 import { TooltipOption } from './TooltipModel';
 
-
-const TOOLTIP_NAME_TEXT_STYLE_CSS = 'font-size:12px;color:#6e7079';
-const TOOLTIP_TEXT_STYLE_RICH = {
-    fontSize: 12,
-    fill: '#6e7079'
-};
-const TOOLTIP_VALUE_TEXT_STYLE_CSS = 'font-size:14px;color:#464646;font-weight:900';
-const TOOLTIP_VALUE_TEXT_STYLE_RICH = {
-    fontSize: 14,
-    fill: '#464646',
-    fontWeight: 900
+type RichTextStyle = {
+    fontSize: number | string,
+    fill: string,
+    fontWeight?: number | string
 };
+
+type TextStyle = string | RichTextStyle;
+
 const TOOLTIP_LINE_HEIGHT_CSS = 'line-height:1';
 
+// TODO: more textStyle option
+function getTooltipTextStyle(
+    textStyle: TooltipOption['textStyle'],
+    renderMode: TooltipRenderMode,
+    tooltipModel?: Model<TooltipOption>
+): {
+    nameStyle: TextStyle
+    valueStyle: TextStyle
+} {
+    const nameFontColor = textStyle.color || '#6e7079';
+    const nameFontSize = textStyle.fontSize || '12px';
+    const valueFontColor = textStyle.color || '#464646';
+    const valueFontSize = textStyle.fontSize || '14px';
+    const valueFontWeight = textStyle.fontWeight || '900';
```

Comment:
It's a bit confusing that only `value` text can change the font-weight. I will prefer both `name` and `value` will be set, just like `fontSize` and `fill`

In the long term, we can discuss a new set of options that can configure each part of the tooltip with enough flexibility. For example the style and formatter of each value.  Of cause that should be a new topic and can be designed in a new pull request.